src/test/java/com/glovoapp/planningpoker/e2e/nametest/SameNameInSessionTest.java

```diff
@@ -0,0 +1,52 @@
+package com.glovoapp.planningpoker.e2e.nametest;
+
+import com.glovoapp.planningpoker.e2e.WebDriverFactory;
+import com.glovoapp.planningpoker.e2e.pages.MainPage;
+import com.glovoapp.planningpoker.e2e.pages.SessionPage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+
+import static com.glovoapp.planningpoker.e2e.WebDriverFactory.getDriver;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class SameNameInSessionTest {
+
+    private final WebDriver driver = getDriver();
+    private final WebDriver driver2 = getDriver();
```

Comment:
Please don't use digits to name multiple variables of same type:

```suggestion
    private final WebDriver secondDriver = getDriver();
```